modules/repository-s3/src/test/java/org/elasticsearch/repositories/s3/RepositoryCredentialsTests.java

```diff
@@ -238,6 +237,10 @@ public void sendResponse(RestResponse response) {
         );
     }
 
+    private Settings.Builder defaultRepoSettings() {
```

Comment:
nit: I'd be inclined to inline this. It's only used in a small number of places, it's not really any shorter like this, and indeed it's a little bit of a distraction to the reader to wonder what exactly those default settings are. It'd be different if there were many required settings ofc.